OptimizelySwiftSDK.podspec

```diff
@@ -12,9 +12,7 @@ Pod::Spec.new do |s|
     :git => "https://github.com/optimizely/swift-sdk.git",
     :tag => "v"+s.version.to_s
   }
-  s.source_files            = "OptimizelySDK/**/*.{swift,h}"
-  s.public_header_files     = "OptimizelySDK/OptimizelySwiftSDK/*.h"
```

Comment:
I think we still need to header files and exclude

Reply:
Can we also fix summary and homepage?
"Optimizely server-side testing core framework."
"Optimizely iOS/tvOS experiment framework" or something like that. :)

Reply:
It looks like header files extracted from the src path by default. We can add the header line when we need support for other paths.